nfl_server/data/career_rushing_stats_10.csv

```diff
@@ -0,0 +1,11 @@
+Pos,Rk,Player,From,To,Draft,Tm,Lg,G,GS,Att,Yds,Y/A,TD,Y/G
+rb,1,Emmitt Smith*,1990,2004,1-17,TOT,NFL,226,219,4409,18355,4.16,164,81.2
```

Comment:
Not sure if we want to keep the `*` beside the player names or not - it just denotes whether or not they are in the Hall of Fame, but we make no mention of it anywhere.